keyboards/giabalanaipico/info.json

```
@@ -0,0 +1,219 @@
+{
+    "keyboard_name": "giabalanaipico",
+    "manufacturer": "3araht",
+    "url": "https://github.com/3araht",
+    "maintainer": "3araht",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "features": {
+        "midi": true,
+        "extrakey": true,
+        "encoder": true,
+        "bootmagic": false,
+        "console": false,
+        "mousekey": false,
+        "nkro": false,
+        "rgblight": false,
+        "audio": false
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "cols": [ "GP22", "GP21", "GP20", "GP19", "GP18", "GP17", "GP16"  ],
+        "rows": [ "GP4",  "GP5",  "GP7",  "GP9",  "GP10", "GP11", "GP12", "GP13", "GP14", "GP15" ]
+    },
+    "debounce": 5,
+    "usb": {
+        "vid": "0xFEED",
+        "pid": "0xF4C0",
+        "device_version": "0.0.1"
+    },
+    "ws2812": {
+        "pin": "GP6",
+        "driver": "vendor"
+    },
+    "encoder": {
+        "rotary": []
+    },
+    "split": {
+        "enabled": true,
+        "soft_serial_pin": "GP0",
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "GP2", "pin_b": "GP3"}
```

migrated from `config.h`
```suggestion
                    {"pin_a": "GP2", "pin_b": "GP3", "resolution": 4}
```

---

I was told that this is a default. I deleted the statement in config.h.